apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -19,14 +19,21 @@ const changeMessage = (rule: Rule, message: string) => {
     return { ...rule, message }
 }
 
-export const useValidations: () => ValidatorTypes = () => {
+type ValidationSettings = {
+    allowLandLine?: boolean;
+}
+
+type UseValidations = (settings?: ValidationSettings) => ValidatorTypes
+
+export const useValidations: UseValidations = (settings) => {
     const intl = useIntl()
     const ThisFieldIsRequiredMessage = intl.formatMessage({ id: 'FieldIsRequired' })
     const PhoneIsNotValidMessage = intl.formatMessage({ id: 'pages.auth.PhoneIsNotValid' })
     const EmailErrorMessage = intl.formatMessage({ id: 'pages.auth.EmailIsNotValid' })
     const FieldIsTooShortMessage = intl.formatMessage({ id: 'ValueIsTooShort' })
     const FieldIsTooLongMessage = intl.formatMessage({ id: 'ValueIsTooLong' })
     const NumberIsNotValidMessage = intl.formatMessage({ id: 'NumberIsNotValid' })
+    const { allowLandLine } = settings ?? {}
```

Comment:
We are trying not to use nullish coalescing and optional chaining as it can break Next.js or in some browsers. There are no cases of using them in the client-side code
```
export const useValidations: UseValidations = (settings = {} ) => {
```

Reply:
fixed